upm-tools/AssemblyDefinitions/BugsnagEditor.asmdef

```diff
@@ -1,7 +1,7 @@
 {
-    "name": "BugsnagEditor",
```

Comment:
This text will never be seen by a user and the actual filename of the assembly is not changing, so please revert this change and leave as it was

Reply:
@clr182 boop